x-pack/plugins/canvas/public/components/palette_picker/palette_picker.js

```diff
@@ -26,6 +26,7 @@ export const PalettePicker = ({ onChange, value, anchorPosition }) => {
       anchorPosition={anchorPosition}
       panelClassName="canvasPalettePicker__swatchesPanel"
       className="canvasPalettePicker__swatchesPopover"
+      anchorClassName="canvasPalettePicker__swatchesPopoverAnchor"
```

Comment:
Just curious, why is this "swatchesPopoverAnchor" instead of just "anchor" like the other one?

Reply:
Because it's a child element of `canvasPalettePicker__swatchesPopover`, so keeping the whole `canvasPalettePicker__swatchesPopover` + `Anchor` is most descriptive.